### ISSUE TITLE
Shot Generator

### ISSUE THREAD

**Issue by Raviranjan-source** [wonderunit/storyboarder#2032]
unable to work in the updated version of Shot Generator.space availability to see the options are too small can't see and use. 
![Screenshot (9)](https://user-images.githubusercontent.com/66679771/84149635-68daea80-aa7e-11ea-85fa-7c983cc85b9c.png)

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by Doolien** [comment]
Yes, I similarly can't access the Properties panel as it gets pushed too far down once multiple assets have been loaded onto the Scene. 
It should really be a floating window.
